packages/entities/src/lib/active/active.spec.ts

```diff
@@ -81,4 +119,45 @@ describe('activeIds', () => {
     store.update(toggleActiveIds(3));
     expect(spy).toHaveBeenCalledWith([{ id: 3, title: '3' }]);
   });
+
+  it('should get the active entities', () => {
+    const { state, config } = createState(
+      withEntities<{ id: number; title: string }>(),
+      withActiveIds()
+    );
+
+    const store = new Store({ state, config, name: '' });
+
+    expect(store.query(getActiveEntities())).toEqual([]);
+
+    store.update(addEntities({ id: 1, title: '' }));
+
+    expect(store.query(getActiveEntities())).toEqual([]);
+
+    store.update(addActiveIds(1));
+
+    expect(store.query(getActiveEntities())).toEqual([{ id: 1, title: '' }]);
+
```

Comment:
Can you please add tests for `ref` and the return type, please?

Reply:
will do, also, I was thinking, would it be a good idea if we could add multiple withActiveId's for different refs (not for this PR)?

Reply:
Yes, we can do with in other PR.

Reply:
separate tests for type and ref or?

Reply:
done